src/plugins/db2sql.c

```diff
@@ -718,21 +718,21 @@ int mobdb2sql_sub(struct config_setting_t *mobt, int n, const char *source)
 		// MEXP
 		StrBuf->Printf(&buf, "%u,", md->mexp);
 
-		for (i = 0; i < 3; i++) {
+		for (i = 0; i < MAX_MVP_DROP; i++) {
```

Comment:
this change affect data fields in sql.
is table header changed to use MAX_MVP_DROP drop too?

Reply:
good call i can expand that side.

Reply:
this should be adjusted now :D